File: Frontend/PlanItRight/src/app/guest.ts
export class Guest {
    id: number;
    name: string;
    email: string;
    rsvpStatus: string;

    constructor(id: number, name: string, email: string, rsvpStatus: string) {
        this.id = id;
        this.name = name;
        this.email = email;
        this.rsvpStatus = rsvpStatus;
    }
}

File: Backend/PlanItRight/EventManagementService/src/main/java/com/PlanItRight/EventManagementService/Model/Guest.java
package com.PlanItRight.EventManagementService.model;


import com.fasterxml.jackson.annotation.JsonIgnore;
import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@AllArgsConstructor
@NoArgsConstructor
@Data
public class Guest {


    @Id
    @GeneratedValue(strategy = GenerationType.SEQUENCE)
    private Long id;

    private String name;
    private String email;
    private String rsvpStatus;
//
//    @ManyToOne(fetch = FetchType.LAZY)
//    @JoinColumn(name = "event_id", nullable = false)
//    @JsonIgnore
//    private com.PlanItRight.EventManagementService.model.Event event;

//    @ManyToOne(fetch = FetchType.LAZY)
//    @JoinColumn(name = "event_id", nullable = false)
//    private Event event;
}


